buildscripts/push

```diff
@@ -92,3 +92,18 @@ else
   echo "No docker credentials provided. Skip uploading ${DIMAGE} to quay";
 fi;
 
+#Push image to run openebs-e2e based on git commit
+function GitlabCommit() {
+  REPO="$1"
+
+  IMAGE_URI="${REPO}:${COMMIT}";
+  sudo docker tag ${IMAGEID} ${IMAGE_URI};
+  echo " push ${IMAGE_URI}";
+  sudo docker push ${IMAGE_URI};
+}
+if [ ! -z "${COMMIT}" ];
+then
+  sudo docker login -u "${DNAMEGL}" -p "${DPASSGL}";
+
+  # Push COMMIT tagged image - :COMMIT
+  GitlabCommit "${DIMAGE}" "${COMMIT}"
```

Comment:
Use the function TagAndPushImage -- instead of using GitlabCommit

Reply:
updated script and removed GitlabCommit function